Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -190,6 +193,11 @@ public async Task ValidateContestPassword(int id, bool official, string password
                     .FirstOrDefault();
             });
 
+            participationModel.ActiveParticipantsCount = this.contestParticipantsCacheService.GetParticipantsCountByContestAndIsOfficial(
```

Comment:
Do you know what "Active" participant means? It's not Official

Reply:
This is no longer relevant.